script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -1319,7 +1319,7 @@ But... It's the honest truth.
 [sha:6ca18e06ec4201ecc2be9af5a8271f938ec9c2fc]{
 -- Page 239, Offset 30197.
 -- 俺はアルクェイドの魔眼であんな事をしたんじゃない。
-It wasn't Arcueid's Mystic Eyes that made me do what I did.
+It wasn't Arcueid's evil eyes that made me do what I did.
```

Comment:
I'm skeptical about this, but it's a strange word to be used instead of your usual 魔眼

Reply:
this is a nasuverse term. revert

Reply:
and the 魔 is just like with 魔剣. It's contextual but in the case of the Nasuverse it is (literally) Magic Eyes/Magic Sword (with "Magic" localised to Mystic to prevent confusion with True Magic/Mahou)

- evil eyes
+ Mystic Eyes